server/container/guice/jpa-common-guice/src/main/java/org/apache/james/modules/data/JPAConfiguration.java

```diff
@@ -65,20 +83,25 @@ public Builder validationQuery(String validationQuery) {
         public JPAConfiguration build() {
```

Comment:
does it make sense to have a password without a username ?

Reply:
Yes, both are linked; you may introduce a `Credential` class for that, IMO.